probeinfo_api.yaml

```diff
@@ -618,6 +630,15 @@ components:
         the BigQuery datasets where decoded ping data is written by the pipeline.
       example: org_mozilla_firefox_beta
 
+    TagFiles:
+      type: array
+      description: A list of relative paths to `tags.yaml` files
+      default: []
+      items:
+        type: string
+        pattern: tags\.yaml$
```

Comment:
https://github.com/mozilla/glean/pull/1849/files#diff-01a82f719b00c6c1cf94ad9ae35864e3997965fd9bcee99a77a4f5f50c3c3b1bR19-R20

We say they can be named whatever, but then probe-scraper actually needs them with a specific name. We should make sure our docs are consistent on that
(I guess our docs on metrics.yaml/pings.yaml are also wrong then, I'll file a bug)

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1738880